JotunnLib/Utils/SimpleJVL.cs

```diff
@@ -258,36 +258,161 @@ public void AddClonedItem(string prefabNew, string prefabOld, string name, strin
+        /// <param name="maxQuality">The maximum amount of upgrades this item can have.</param>
+        /// <param name="maxStackSize">Maximum stack size this item can have.</param>
+        /// <param name="movementModifier">The movement speed increase, as a multiplier.</param>
+        /// <param name="questItem">Sets whether or not this item treated as a quest item.</param>
+        /// <param name="spawnOnHit"><see cref="GameObject"/> spawned on successful hit.</param>
+        /// <param name="spawnOnHitTerrain"><see cref="GameObject"/> spawned on successful hit on terrain.</param>
+        /// <param name="teleportable">Sets whether or not this item can be brought through portals.</param>
+        /// <param name="value">Value of this item to Haldor, the Trader.</param>
+        /// <param name="variants">Number of variants of this item during crafting.</param>
+        /// <param name="weight">Weight of the item, default scaling is ~5:1kg. Default carry weight of the player is 300, or ~60kg.</param>
+        /// <param name="blockable">Whether or not this item's attacks are blockable, or whether they ignore "blocking via right click".</param>
+        /// <param name="blockPower">Value of "blocking via right click", for this item.</param>
+        /// <param name="blockPowerPerLevel">Additional value of <paramref name="blockPower"/>, for this item; per upgrade of this item.</param>
+        /// <param name="equipDuration">Length of time that equipping this item takes, in seconds.</param>
+        /// <param name="equipStatusEffect">Triggered <see cref="StatusEffect"/> on equip of this item.</param>
+        /// <param name="consumeStatusEffect">Triggered <see cref="StatusEffect"/> on consuming this item.</param>
+        /// <param name="food">Value added to Hungerbar upon consumption.</param>
+        /// <param name="foodBurnTime">Length of time this item gives it's bonuses for.</param>
+        /// <param name="foodRegen">How much Health this item grants per second.</param>
+        /// <param name="foodStamina">How much Stamina this item grants per second.</param>
+        /// <param name="setName">Name of the set that this item belongs to.</param>
+        /// <param name="setSize">Size of the set that this item belongs to.</param>
+        /// <param name="setStatusEffect">Triggered <see cref="StatusEffect"/> when player has equipped <paramref name="setSize"/> of <paramref name="setName"/></param>
+        /// <param name="durabilityDrain">Durability reduced for each use.</param>
+        /// <param name="durabilityPerLevel">Additional <paramref name="durabilityDrain"/> per upgrade of this item.</param>
+        /// <param name="useDurability">Durability reduced for each use.</param>
+        /// <param name="useDurabilityDrain">Whether or not this item uses <paramref name="durabilityDrain"/> when actions are performed.</param>
+        public void AddClonedItem(string prefabNew, string prefabOld, string name, string description,
```

Comment:
The method of Mona Lisa ;)

Reply:
Hey, I considered it thoroughly before implementing it that way. If the constructor has 1 overload that points to this, the user doesn't get the spam from IntelliSense, but functionality stays the same, additionally allows the author to target specific fields while leaving others untouched simply by adding on, which creates the overload situation into the optional params. I couldn't find a better way to get this kind of wrapping, but I am open to suggestions.

Reply:
Removed _The Methods of Mona Lisa_ until further notice.